Automattic/vip-go-mu-plugins
# Search: Change warning for accuracy to use correct force_parallel_execution option

## Description
When we try to run two instances of `wp vip-search health validate-contents` at once, the warning gives the incorrect option to use:

```
Error: Content validation is already ongoing

You can use --force-parallel-execution to run the command even with the lock in place
Error: You can use --force-parallel-execution to run the command even with the lock in place
```

It should be with underscores as defined by the synopsis: https://github.com/Automattic/vip-go-mu-plugins/blob/ee62537d2cd30ed4defe7c74d4a1d155c042e06e/search/includes/classes/class-health.php#L413-L414

## Changelog Description

### Search: Change warning for accuracy to use correct force_parallel_execution option

Instead of the option with underscores: `--force-parallel-execution`

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Attempt to run `wp vip-search health validate-contents` more than once.
2) See error message reference incorrect option
3) Check out PR
4) Repeat step 1) and see expected results
